packages_dev/all/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "private": true,
+  "name": "@decidim/all",
+  "description": "All dependencies for Decidim",
+  "version": "0.25.0-dev",
+  "repository": {
+    "url": "git@github.com:decidim/decidim.git",
+    "type": "git",
+    "directory": "packages/all"
+  },
+  "author": "Decidim Contributors",
+  "license": "AGPL-3.0",
+  "engines": {
+    "node": "^15.14.0",
+    "npm": "^7.7.2"
+  },
+  "dependencies": {
+    "@decidim/browserslist-config": "https://gitpkg.now.sh/mainio/decidim/packages/browserslist-config?feature/split-npm-packages",
```

Comment:
💬 Can we remove the branch reference here? Or do you prefer to create a new PR to change it after merging this?

Reply:
Yes, after this is merged (in another PR).